core/src/Cabana_LinkedCellList.hpp

```diff
@@ -222,6 +268,29 @@ class LinkedCellList
     KOKKOS_INLINE_FUNCTION
     std::size_t rangeEnd() const { return _bin_data.rangeEnd(); }
 
+    /*!
+      /bried Given a cell, get the index bounds of the cell stencil
+    */
+    KOKKOS_INLINE_FUNCTION
+    void getCells( const int cell, int& imin, int& imax, int& jmin, int& jmax,
```

Comment:
Not sure exactly how, but we need to decide how to differentiate this from `getBins`

Reply:
Maybe renaming this to `getStencilCells` and `getBins` to `getParticleBins`? I think those names would be better descriptions of what the functions are actually doing while also being differentiable from each other

Reply:
I like those names - still a bit unsure of whether we should just call everything "bin" instead of "cell". But stencil vs particle is helpful no matter what